.circleci/config.yml

```diff
@@ -69,20 +69,6 @@ jobs:
       - gomod
       - run: go test -v -race ./...
 
-  test_cover:
```

Comment:
Do we need other circle ci actions?

Reply:
I needed this to ensure that the new Coverage workflow works well and publishes the results to the PR page.

I think that we're not planning to remove circleci actions at the current moment. Let's keep them both for a while, compare and decide which one we'd better get rid of.